### ISSUE TITLE
Implement chrome page monitor in adhoc browser pool

### ISSUE THREAD

**Issue by wzin** [EndPointCorp/lg_ros_nodes#131]
tl;dr this ROS node should make it possible to lock users inside an experience (like google maps). It's often outside of our power to remove all offsite links from experiences that we run on ROScoe.

Requirements:
- make a parameter liek `monitor_page_url`
- provide list of allowed urls regexps with wildcard using numbered list of urls liek `allowed_url_1=http://localhost:8088/*` and `allowed_url_2=http://localhost:8089/*`
- above list should be provided in `activity_config` attribute
- check chrome list of tabs periodically (parametrize it) to check if the URL matches the regexp

Development notes:
- probably implement it on ManagedBrowser level 
- port portion of lg-browser from portal here (chromium-page-monitor actually)

**Comment by axisofentropy** [comment]
On 12/18/2015 12:05 PM, Wojciech Ziniewicz wrote:
>   * check chrome list of tabs periodically (parametrize it) to check if
>     the URL matches the regexp

Would this check tab crashes? Does that change the URL?

**Comment by ezekiel** [comment]
When the tab crashes, it simply disappears from the "chrome list of tabs" being checked periodically. So this _will_ catch tab crashes if at least one of the allowed URLs is required to be matched AND only one tab is opened per browser instance.

In python, for example, _requiring_ the "set" iterable returned by:
```
<current_page_urls>.intersection(<allowed_urls>)
```
to be non-empty, would catch tab crashes where only one tab is open.

This _will_NOT_ catch tab crashes otherwise.
